tests/utils/utils.php

```diff
@@ -41,18 +42,51 @@ function get_class_method_as_public( $class_name, $method ) {
 	$reflector = new ReflectionClass( $class_name );
 	$method    = $reflector->getMethod( $method );
 
-	$method->setAccessible( true ); // NOSONAR
+	$method->setAccessible( true ); // NOSONAR.
 
 	return $method;
 }
 
 /**
- * @psalm-param class-string $class_name
+ * For testing purpose gets private static property of a class by making it as public.
+ *
+ * @param string $class_name Name of the class.
+ * @param string $property_name Name of the property.
+ *
+ * @return ReflectionProperty
  */
 function get_static_property_as_public( string $class_name, string $property_name ): ReflectionProperty {
 	$reflection = new ReflectionClass( $class_name );
 	$property   = $reflection->getProperty( $property_name );
-	$property->setAccessible( true ); // NOSONAR
+	$property->setAccessible( true ); // NOSONAR.
 
 	return $property;
 }
+
+/**
+ * Setup custom error handler and returns current level of error reporting.
+ */
+function setup_custom_error_reporting(): int {
```

Comment:
Moved duplicated code to following methods

- setup_custom_error_reporting
- reset_custom_error_reporting